examples/eady_turbulence.jl

```diff
@@ -60,18 +60,22 @@
 # the flux is negative (downwards) when the velocity at the bottom boundary is positive, and 
 # positive (upwards) with the velocity at the bottom boundary is negative.
 # This drag term is "quadratic" because the rate at which momentum is removed is proportional
-# to $\bm{u}_h \sqrt{u^2 + v^2}$, where $\bm{u}_h = u \bm{\hat x} + v \bm{\hat{v}$ is the
-# horizontal velocity.
+# to $\boldsymbol{u}_h |\boldsymbol{u}_h|$, where 
+# $\boldsymbol{u}_h = u \boldsymbol{\hat{x}} + v \boldsymbol{\hat{y}}$ is the horizontal velocity.
```

Comment:
this is fine and I will merge, but just out of curiosity why boldsymbol vs bm?

Reply:
hm... I thought `\bm` was part of the problem for not latexifying... perhaps I was wrong?

Reply:
I think your math block fixed the main problem but I could be wrong. Used to be that KaTeX / Documenter didn't support `\boldsymbol` (_only_ `\bm` was supported). Seems that's no longer the case, though if I had to guess, I would guess that they are aliases for each other. I believe `\bm` is supposed to be some kind of simple, lightweight version of `\boldsymbol`.

Reply:
The above assertion can be tested... but I'll leave that for some future time :)

Reply:
They might be aliases in KaTeX/Documenter but I think `bm` is a stand-alone package in vanilla LaTeX: https://ctan.org/pkg/bm?lang=en